src/MagicalCards.elm

```diff
@@ -1,4 +1,101 @@
 port module MagicalCards exposing (..)
-import Html exposing (Html, text)
 
-main = text "Hello World!"
+import Html exposing (..)
+import Html.Events exposing (..)
+import Library.Library exposing (..)
+import Hand.Hand exposing (..)
+import List exposing (..)
+
+
+main =
+    Html.program
+        { init = init
+        , view = view
+        , update = update
+        , subscriptions = subscriptions
+        }
+
+
+
+-- MODEL
+
+
+type alias Library =
+    Library.Library.Library
+
+
+type alias Hand =
+    Hand.Hand.Hand
+
+
+type alias Model =
+    { library : Library
+    , hand : Hand
+    }
+
+
+init : ( Model, Cmd Msg )
+init =
+    ( Model (Library.Library.init 5) Hand.Hand.init, Cmd.none )
+
+
+
+-- UPDATE
+
+
+type Msg
+    = LibraryMsg
+    | HandMsg
+    | DrawCard
+
+
+update : Msg -> Model -> ( Model, Cmd Msg )
+update msg model =
+    case msg of
+        LibraryMsg ->
+            ( model, Cmd.none )
+
+        HandMsg ->
+            ( model, Cmd.none )
+
+        DrawCard ->
+            ( drawCard model, Cmd.none )
+
+
+drawCard : Model -> Model
+drawCard model =
```

Comment:
That's a good point. I suspect in the future we will have a couple different "glue" modules so our top level does not become too unwieldy.